pkg/mounter/proxy/server/ossfs2/driver.go

```diff
@@ -73,9 +74,13 @@ func (h *Driver) Mount(ctx context.Context, req *proxy.MountRequest) error {
 	args = append(args, "-f")
 
 	var stderrBuf bytes.Buffer
+	sw := serverutils.NewSwitchableWriter(&stderrBuf)
 	cmd := exec.Command("ossfs2", args...)
 	cmd.Stdout = os.Stdout
-	cmd.Stderr = &stderrBuf
+	cmd.Stderr = sw
+	defer func() {
+		sw.SwitchTarget(os.Stderr)
```

Comment:
Use `io.MultiWriter(&stderrBuf, os.Stderr)` as initial target to avoid losing any output.